packages/widget-v2/src/pages/SwapPage/useSetMaxAmount.ts

```diff
@@ -0,0 +1,75 @@
+
+import { convertHumanReadableAmountToCryptoAmount, convertTokenAmountToHumanReadableAmount } from "@/utils/crypto";
+import { useGetAssetDetails } from "@/hooks/useGetAssetDetails";
+import { sourceAssetAmountAtom, sourceAssetAtom } from "@/state/swapPage";
+import { useAtom, useSetAtom } from "jotai";
+import { skipChainsAtom } from "@/state/skipClient";
+import { useSourceBalance } from "./useSourceBalance";
+import { BigNumber } from "bignumber.js";
+
+
+const ETH_GAS_FEE = 0.1;
```

Comment:
this should be 0.01 or 10 * 10 ** 16

Reply:
I think it's a little easier to understand if we keep the value in terms of how much ETH and then convert it to the tokenAmount using a util like this:

`convertHumanReadableAmountToCryptoAmount(ETH_GAS_FEE)`

but I do need to pass the correct decimals which i forgot to do